### PR TITLE
List orgs

### LINKED ISSUE #71: Create api to list the orgs involved by the user
**Is your feature request related to a problem? Please describe.**
As a user i would like to see the list of orgs I am a member.

image

**Describe the solution you'd like**
1. Create an api: `[get] /api/user/org/` which lists the orgs I am a member of.
1. only authenticated user can access it.
1. make swagger docs for the same 
1. add tests for the same

### PR DESCRIPTION
**Pull request by anshsrtv**
### Description
List Organisations API which provides the list of organisations, the user is a member of, is ready. The swagger documentation is updated and the API has been tested manually as well as by django tests

**Type of Change:** (Delete irrelevant options)
- Code
- Testing
- Documentation

**Code/Quality Assurance Only** (Delete irrelevant options)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
This has been tested locally using Postman, using swagger docs and by the django tests mechanism.
 **Unauthenticated requests:**

**Authenticated requests:**

### Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged
- [x] Update Swagger documentation and the exported file at /docs folder
- [ ] Update requirements.txt

**Code/Quality Assurance**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: users/views.py
Comment (naveennvrgup) [inline users/views.py]: please also return
1. profile_pic 
1. id
1. route_slug
1. tagline

File: users/views.py
Comment (naveennvrgup) [inline users/views.py:112]: Please add an API description also :smile_cat: 

image

**Review by naveennvrgup — CHANGES_REQUESTED** [review]
1. Tested locally & works as expected :clap: .
2. Please address my comments.

File: users/views.py
Comment (anshsrtv) [reply users/views.py]: Whose id and route_slug?

Comment (naveennvrgup) [reply users/views.py]: @anshsrtv  areee Org ka hai

Comment (anshsrtv) [reply users/views.py]: @naveennvrgup Made the changes .

**Review by naveennvrgup — APPROVED** [review]
1. Tested locally works as expected
1. can be merged after 2 pr approvals

**Comment by sahil9001** [comment]
@naveennvrgup @anshsrtv Can you return `{ "message" : "User has no organisations" }` if the user is not a part of any organisation? Currently it only shows an empty object `[]`

**Comment by naveennvrgup** [comment]
> @naveennvrgup @anshsrtv Can you return `{ "message" : "User has no organisations" }` if the user is not a part of any organisation? Currently it only shows an empty object `[]`

An empty object will be fine in the frontend. Otherwise, they have to add some if-else conditions for this scenario. So let's keep it simple.

**Comment by sahil9001** [comment]
Okay!

**Review by sahil9001 — APPROVED** [review]

**Pull request merged**
